webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -351,6 +352,107 @@ def test_if_get_and_full_data_from_session_then_delete_unnecessary_data(self):
                              deserialize_session_data(req.session[_ELIGIBILITY_DATA_KEY]))
 
 
+class TestSeparatedLivedTogetherEligibilityInputFormSteuerlotseStep(unittest.TestCase):
```

Comment:
Shouldn't we also test `SeparatedJointTaxesEligibilityInputFormSteuerlotseStep`? And shouldn't we also adapt the test data for the other tests?

Reply:
You are completely right, I forgot the tests for `SeparatedJointTaxesEligibilityInputFormSteuerlotseStep`. (it just already looked like a lot. :x)

I don't think I get what you mean by 
> And shouldn't we also adapt the test data for the other tests?
though. If you mean that I didn't add a second test case to `SingleAlimony` althought it now has a new previous step, you are right and I'll fix it. Please explain if you mean something else though.

Reply:
This was addressed in #103